### PR TITLE
SPA - Mastery Exercise

### PR DESCRIPTION
**Pull request by daynewright**

### CODE REVIEW

File: carLot.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+      inventoryLoader.open('GET', 'inventory.json');
+      inventoryLoader.addEventListener("load", function (evt) {
+        inventory = JSON.parse(evt.target.responseText).cars;
+        cb(inventory);
+      });
+      inventoryLoader.send();
+    },
+
+    activateEvents: function() {
```
Comment (gregkorte) [inline carLot.js]: This function should be in it's own augmented IIFE as per the instructions.

File: carLot.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        document.querySelector('input').removeEventListener('keyup', changeDescript);
+        document.querySelectorAll('.car-block').forEach(e => e.style = `border: 3px solid ${e.style.borderTopColor}`);
+
+        this.style.border = `5px solid ${evt.currentTarget.style.borderTopColor}`;
+        this.style.background = "#e6e6e6";
+        input.focus();
+        input.value = "";
+
+
+        descript = evt.currentTarget.childNodes[11];
```
Comment (gregkorte) [inline carLot.js]: Careful when digging into these DOM objects this way. It makes your code rigid and fragile. What happens if you want to change the way the card looks? Will it break your app?

File: carLot.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    activateEvents: function() {
+      var carBlock = document.querySelectorAll('.car-block');
+      var input = document.querySelector('input');
+      var descript;
+
+      carBlock.forEach(e => e.addEventListener('click', changeBlock));
+      input.addEventListener('keypress', updateDescription);
+
+      function changeBlock(evt){
```
Comment (gregkorte) [inline carLot.js]: This function should be augmented in it's own IIFE as per the instructions. It should also take the DOM element that was clicked on and the color.

**Comment by gregkorte** [comment]
Great job on your quiz, Dayne! Break out those few pieces of code into their own augmented files, and you will be good. Overall I like the way you pared the code down to two files. It's clean and efficient. However, if you take another look at the technical instructions it will give you some context to my comments. Let me know via slack when you have your changes pushed, and I'll take another look.

**Comment by gregkorte** [comment]
You are good to merge! :+1:

File: carLot.js (hunk 2 above)
Comment (daynewright) [inline carLot.js]: I am going to try to mess with this tonight before merging.  Will see if I can figure it out.

**Pull request merged**